README.md

```diff
@@ -22,12 +22,30 @@ languages.
  - https://www.php.net/mongodb
  - https://www.mongodb.com/docs/drivers/php-drivers/
 
-## Installation
+## Installation with pie
+
+To install this extension, you need [pie](https://github.com/php/pie) installed on your system. `pie` is a modern tool for managing PHP extensions.
+
+Install the [`mongodb/mongodb-extension`](https://packagist.org/packages/mongodb/mongodb-extension) package from Packagist using the following command:
+
+```shell
+pie install mongodb/mongodb-extension
```

Comment:
We should add an example on how to use version constraints - we could show how to install version 2.1 or newer:
```shell
pie install "mongodb/mongodb-extension:^2.1"
```